orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -79,6 +80,9 @@ public class DetermineRollbackCandidatesTask implements CloudProviderAware, Retr
   private static final TypeReference<List<ServerGroup>> listOfServerGroupsTypeReference =
       new TypeReference<>() {};
 
+  @Value("${rollback.timeout.enabled:false}")
+  private static boolean stageRollbackTimeout;
```

Comment:
doesn't need to be static. you can add a new argument to the existing constructor
```
private final boolean dynamicRollbackTimeoutEnabled;

  @Autowired
  public DetermineRollbackCandidatesTask(
      ObjectMapper objectMapper,
      RetrySupport retrySupport,
      CloudDriverService cloudDriverService,
      FeaturesService featuresService,
      @Value("${rollback.timeout.enabled:false}") boolean dynamicRollbackTimeoutEnabled) {
    this.retrySupport = retrySupport;
    this.cloudDriverService = cloudDriverService;
    this.previousImageRollbackSupport =
        new PreviousImageRollbackSupport(
            objectMapper, cloudDriverService, featuresService, retrySupport);
    this.dynamicRollbackTimeoutEnabled = dynamicRollbackTimeoutEnabled;
  }
  ```